mbspotify/views_test.py

```diff
@@ -56,8 +56,8 @@ def create_app(self):
 
     def test_index(self):
         response = self.client.get("/")
-        # Index page should ask users to piss off.
-        self.assertIn("Piss off!", str(response.data))
+        # Index page should redirect visitors to the project page on GitHub.
+        self.assertRedirects(302, "https://github.com/metabrainz/mbspotify")
```

Comment:
You might be able to check the status separately. Check docs for that function at https://pythonhosted.org/Flask-Testing/.